### DISCUSSION TITLE
Abort due to 'Segamentation Fault'

### DISCUSSION THREAD

**Discussion by YuanZhongYan** [MPh-py/MPh#181] [General]
Hi there,

This is a wonderful Python API for COMSOL; thanks to the developers.

I have used the MPH package to call COMSOL for random simulations for a while. The script works well with version 1.2.3. However, PyCharm randomly raises the error: **Magick: abort due to signal 11 (SIGSEGV) "Segmentation Fault"...** after I updated the package to the newest version (1.2.4). Is there something wrong with my environment setup?

**Comment by john-hen** [comment]
Hi. Version 1.2.4 contains only trivial changes: better error messages. Magick, which I believe refers to ImageMagick, is an external dependency of Comsol. I don't know why you get that segmentation fault all of a sudden, but the new MPh version can't be the reason. Nothing changed in that regard.

**Reply by YuanZhongYan** [reply]
Thank you for the quick response. 
Yes, I see the explanations regarding version 1.2.4. It is possible that some Python versions may also raise that error. I will try downgrading the Python version to check whether the script can run stably.
